docker/all-in-one/healthcheck.sh

```diff
@@ -37,5 +37,7 @@ fail2ban-client status
 # prometheus exporter up
 curl -sSfI "http://localhost:$PGEXPORTER_PORT/metrics"
 
-# vector is up
-curl -sSfI "http://localhost:$VECTOR_API_PORT/health"
+# vector is up (if starting logflare)
+if [ -n "${LOGFLARE_API_KEY:-}" ]; then
+  curl -sSfI "http://localhost:$VECTOR_API_PORT/health"
+fi
```

Comment:
We don't currently spin up a local Logflare for Vector tests, but we still need the healthcheck to pass before we run testinfra tests.

Reply:
Should add a TODO to remove this guard once logflare is spun up; adding this guard here makes the setup more fragile